projects/canopy/src/lib/canopy.module.ts

```diff
@@ -21,6 +21,7 @@ import { LgIconModule } from './icon/icon.module';
 import { LgPageModule } from './page/page.module';
 import { LgPipesModule } from './pipes/pipes.module';
 import { LgPrefixModule } from './prefix/prefix.module';
+import { LgPromoCardModule } from './promo-card';
```

Comment:
```suggestion
import { LgPromoCardModule } from './promo-card.module';
```
There is an issue  in ng-packagr where you need to add the file extension to module imports.